README.md

```diff
@@ -1,36 +1,28 @@
 # Taskcluster Component Loader
 
-Creates a component loader from a set of inter-dependent component definitions.
+This library provides a means of loading application "components", each of
+which can depend on other components.  This makes application startup more
+modular and flexible.  It also enables dependency injection during tests.
+
+It is used to run all TaskCluster microservices.
+
+## Components
+
 Each component definition specifies:
+
   * Name of the component,
   * Required components to be instantiated first, and,
   * How to asynchronously load the component.
 
-Given a set of these definitions, `taskcluster-lib-loader` will ensure that
-definitions are valid, that dependencies forms a directed acylic graph (DAG),
-and return a method `load(componentName, overwrites)` which will asynchronously
-load a component by `componentName`. Before loading the component it will
-load all dependent components not specified in the `overwrites` dictionary.
+All of this is specified as properties of the components object.  A server
```

Comment:
(I think I did that below in the "Loading" section - let me know if I got it wrong)